spec/lib/cms/indexers/page_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe Indexers::Page do
+  it_behaves_like 'an indexer', 'pages'
+
+  subject(:page_indexer) do
+    described_class.new(collection: collection, adapter: adapter)
+  end
+
+  let(:adapter) { 'local' }
+
+  describe '#objects' do
+    let(:collection) do
+      [
+        create(:page,
+               label: 'Financial well being',
+               slug: 'financial-well-being',
+               meta_description: 'description',
+               published_at: Date.today,
```

Comment:
not a blocker, but FYI - using the latest rubocop *and* the yaml config from mas-standards rubocop would raise the following offense
```
page_spec.rb:17:35: C: Rails/Date: Do not use Date.today without zone. Use Time.zone.today instead.
```